Elasticsearch: base tests

kvignesh1420 opened the pull request:
This PR addresses the feature request: https://github.com/tensorflow/io/issues/1093 by adding the base test cases.

terrytangyuan commented on tests/test_elasticsearch_eager.py:
```diff
@@ -0,0 +1,127 @@
+    assert issubclass(type(dataset), tf.data.Dataset)
+
+    for item in dataset:
+        for attr in ["name", "gender", "age", "fare", "survived"]:
```
Move this list to a constant since it appears in multiple places

kvignesh1420 replied:
addressed in the commit

kvignesh1420 commented:
@terrytangyuan I switched to using sockets for checking the presence of the es service as docker was posing trouble. Also, moved the `ATTRS` to a common var

terrytangyuan reviewed (approved):
LGTM. Others PTAL

yongtang reviewed (approved):
LGTM
